server/service/osquery_utils/queries.go

```diff
@@ -555,17 +555,11 @@ var extraDetailQueries = map[string]DetailQuery{
 		DirectIngestFunc: directIngestChromeProfiles,
 		Discovery:        discoveryTable("google_chrome_profiles"),
 	},
-	"battery_macos": {
```

Comment:
Does removing these query key have a BC impact on existing Fleet server installs?

Reply:
this change actually reverts yesterdays commit (`battery` -> `battery_macos` + `battery_windows`), but either way there shouldn't be any impact.